techniques/systemSettings/process/category.xml

```diff
@@ -16,5 +16,5 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 <xml>
     <name>Process</name>
```

Comment:
This should be renamed "Services" if possible, since the only visible technique in it is also about Services.